### PR TITLE
WIP Feature: VAM Preset Manager

### PR DESCRIPTION
**Pull request by linkion**

### CODE REVIEW

File: 3denEnhanced/GUI/VAMPresetsMenu.hpp
Comment (hypoxia125) [inline 3denEnhanced/GUI/VAMPresetsMenu.hpp:1]: Consider changing spacing to 2 spaces to match the rest of the mod.

File: 3denEnhanced/functions/GUI/VAM/fn_VAM_openPresetsMenu.sqf
Comment (hypoxia125) [inline 3denEnhanced/functions/GUI/VAM/fn_VAM_openPresetsMenu.sqf:1]: same spacing suggestion as before (2 spaces)

**Comment by linkion** [comment]
Now ready for review!

**Comment by linkion** [comment]
there

thank you GitHub desktop

**Comment by linkion** [comment]
@R3voA3 This is good to go!

**Pull request merged**